src/commands/isCASDown.ts

```diff
@@ -0,0 +1,42 @@
+import { EmbedBuilder, SlashCommandBuilder } from 'discord.js';
+import axios from 'axios';
+
+const statusURI = 'https://cas.byu.edu/cas/serviceValidate';
+
+const statuses = {
+	down: {
+		message: 'CAS is down again.',
+		color: 0xdc2626,
+	},
+	up: { message: 'CAS is running, for now.', color: 0x16a34a },
+};
+
+const builder = new SlashCommandBuilder()
+	.setName('iscasdown')
+	.setDescription('Checks if CAS is down');
+
+export const isCASDown: GlobalCommand = {
+	info: builder,
+	requiresGuild: false,
+	async execute({ reply }) {
+		const embed = new EmbedBuilder().setTitle('Is CAS Down?');
+
+		let status = statuses.down;
+
+		const req = await axios.get(statusURI).catch(() => null);
```

Comment:
Network errors might be useful to know. Also, our command runner architecture knows what to do with them, so best to let the throw happen.

```suggestion
		const req = await axios.get(statusURI);
```

Reply:
Also, a network error doesn't necessarily mean that CAS is down. We'd expect something like a successful response with a status of `500`. So catching the error and defaulting to a `null` response wouldn't be useful.

Reply:
I made this change. Didn't realize that there was handling of it, but my thought was if there was an error, the command still needs to report it as CAS being down, so having it be `null` would trigger that. We could always revert though.